jwql/tests/test_utils.py

```diff
@@ -175,6 +177,35 @@ def test_filename_parser(filename, solution):
     assert filename_parser(filename) == solution
 
 
+@pytest.mark.xfail(reason='Known non-compliant files in filesystem')
+def test_filename_parser_whole_filesystem(capsys):
+    """Test the filename_parser on all files currently in the filesystem.
+    """
+    # Get all files
+    filesystem_dir = get_config()['filesystem']
+    all_files = []
+    for dirName, _, fileList in os.walk(filesystem_dir):
+        for file in fileList:
+            if file.endswith('.fits'):
```

Comment:
@laurenmarietta Good idea to check for `.fits` files.  There will probably come a time when rando MAST files are in the filesystem.